documentation/assemblies/overview/assembly-metrics-overview.adoc

```diff
@@ -35,3 +35,4 @@ include::modules/con-metrics-overview-prometheus.adoc[leveloffset=+1]
 include::modules/con-metrics-overview-grafana.adoc[leveloffset=+1]
 include::modules/con-metrics-overview-exporter.adoc[leveloffset=+1]
 include::modules/con-metrics-overview-tracing.adoc[leveloffset=+1]
+include::../../modules/cruise-control/con-cruise-control-overview.adoc[leveloffset=+1]
```

Comment:
I think this should include `con-metrics-overview-cruisecontrol.adoc`. 

Are you going to provide metrics-specific content for Cruise Control in a later PR? If not, is it necessary to include a metrics module for Cruise Control? At the moment, it feels like a bit of a filler.

Reply:
I removed it for now

Reply:
OK.